sfn-query-large-datasets-cdk/source.bat

```diff
@@ -0,0 +1,13 @@
+@echo off
```

Comment:
There is no reference to this file in the README. Is it needed? Apologies if I am missing some python related stuff.

Reply:
it's the windows default source.bat from cdk init. It's included in all other ones as-well.